hack/kccm-flavor-gen.sh

```diff
@@ -5,6 +5,12 @@ set -x -e -o pipefail
 kccm_template=/tmp/kccm.yaml
 
 kubectl kustomize config/kccm > $kccm_template
+
+
+sed -i \
+    -E "s|(namespace: )kvcluster|\1\${NAMESPACE}|g;s|(^.*cluster-name=)kvcluster|\1\${CLUSTER_NAME}|g" \
```

Comment:
Do you know if we can parameterize it at cloud-provider-kubevirt with something like https://github.com/cert-manager/cert-manager/blob/f970eca76265ff22c868dab3699cdd3cae26e90f/deploy/charts/cert-manager/templates/webhook-deployment.yaml#L68-L69